content-src/experiments/tracking-protection.yaml

```diff
@@ -86,6 +86,7 @@ contributors:
     display_name: 'Rebecca Billings'
     title: 'Senior QA Engineer'
     avatar: /static/images/experiments/tracking-protection/avatars/rebecca-billings.jpg
+contributors_extra: 'This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years. Learn more about <a href="https://mzl.la/2fYneDq" target="blank">Firefox Tracking Protection</a>.'
```

Comment:
Why it's a bad idea:
* You put something that should not be localized in a localizable string. It makes the string more confusing (because of markup), easier to break, potentially dangerous.
* If you need to update the URL, you would need to change the entire string, while the actual string doesn't change at all.